test/celeritas/em/CoulombScattering.test.cc

```diff
@@ -370,28 +424,33 @@ TEST_F(CoulombScatteringTest, simple_scattering)
 
 TEST_F(CoulombScatteringTest, distribution)
 {
-    CoulombScatteringHostRef const& data = model_->host_ref();
+    auto const material = this->material_track().make_material_view();
+    IsotopeView const isotope
+        = material.make_element_view(elcomp_id_).make_isotope_view(isocomp_id_);
+
+    // TODO: Use proton ParticleId{2}
```

Comment:
Just curious where this proton cut used? Is this cut used for the production recoiled nucleus/ions?

Reply:
Referring back to https://github.com/celeritas-project/celeritas/pull/861#discussion_r1288852391